### ISSUE TITLE
wechat@3.9.11.25: hash check failed

### ISSUE THREAD

**Issue by PIPIPIG233666** [ScoopInstaller/Extras#14020]
### Prerequisites

- [X] I have used the predefined issue title. (e.g. "xxx@xxx&#58; hash check failed")
- [X] I have verified that I am using the latest version of Scoop and corresponding bucket.

### Package Name and Version

wechat@3.9.11.25

**Comment by PIPIPIG233666** [comment]
```
ERROR Hash check failed!
App:         extras/wechat
URL:         https://dldir1v6.qq.com/weixin/Windows/WeChatSetup.exe#/dl.7z
First bytes: 4D 5A 90 00 03 00 00 00
Expected:    04a668ea0d36cd8963acc2b9c0f0aff0c0a7c73f46127863ea1f90e942664171
Actual:      76b4d0f4dc28a679bea09b2471444956e43680982b31eec9584e9ccf695046d9
```

**Comment by aliesbelik** [comment]
Fixed by #14022.
